test/likelihoods/test_gaussian_likelihood.py

```diff
@@ -84,6 +85,13 @@ def test_fixed_noise_gaussian_likelihood(self, cuda=False):
             obs_noise = 0.1 + torch.rand(5, device=device, dtype=dtype)
             out = lkhd(mvn, noise=obs_noise)
             self.assertTrue(torch.allclose(out.variance, 1 + obs_noise))
+            # test noise smaller than min_fixed_noise
+            expected_min_noise = settings.min_fixed_noise.value(dtype)
+            noise[:2] = 0
+            lkhd = FixedNoiseGaussianLikelihood(noise=noise)
```

Comment:
Should we also check that the warning is raised here? Not super critical though.